src/interactive/Dropdown.jsx

```diff
@@ -50,9 +50,6 @@ type Props = {
 	/** Props to pass to the `Downshift` component */
 	downshiftProps?: Object,
 
-	/** Whether the dropdown should close when a click on the body is registered */
-	closeOnBodyClick?: boolean,
```

Comment:
if we are removing this its  a breaking change as i believe this is a breaking change in mupweb which is fine, we just need to note that in the version of this by incrementing the version # accordingly and when this upgrade happens in mupweb handle the removal :)